codegen/Package.swift

```diff
@@ -70,7 +70,7 @@ func appendLibTarget(name: String, path: String) {
                     ),
                     .product(
                         name: "AWSClientRuntime",
-                        package: "AWSClientRuntime"
+                        package: "AWSSwiftSDK"
```

Comment:
this seems incorrect. Why AWSSwiftSDK is not defined as package.

Reply:
This is indeed correct.

Instead of being a product published from its own package, i.e. package=`AWSClientRuntime` & product=`AWSClientRuntime`, `AWSClientRuntime` is now a product published from the `AWSSwiftSDK` package, i.e. package=`AWSSwiftSDK` and product=`AWSClientRuntime`.

This is similar to how the service clients are products published by the `AWSSwiftSDK` project.  The only difference here is that `AWSClientRuntime` is not auto-generated.

As proof that this configuration is correct, if you generate files and run `swift test` from within the `codegen/` directory, it resolves dependencies & runs tests successfully.

Reply:
awesome, thanks for verifying it.